examples/isomorphic-flux-chat/js/main.js

```diff
@@ -0,0 +1,25 @@
+var mockData = require('./mock-data')
+var Nuclear = require('nuclear-js')
+var reactor = new Nuclear.Reactor({
+  debug: true,
+})
+window.reactor = reactor
+var Chat = require('./modules/chat')
+
+var ChatApp = require('./components/ChatApp.react')
+ChatApp = Nuclear.provideReactor(ChatApp)
```

Comment:
Slightly O/T idea: what if `Nuclear.Reactor` had a `registerActions` method, so you could call something more like `this.context.reactor.clickThread(payload)` ? Or, rather than using the `initialize / this.on` pattern, Stores had an `actions` obj: 

```js
const myStore  = Nuclear.Store({
  actions: {
    clickThread: (currentState, payload) => currentState.set(payload.id, payload),
  },
  getInitialState() { 
    return Immutable.Map()
  }
})

then, when you do `reactor.registerStores()`, Nuclear crawls all `store.actions`s, and "merges" dupes (eg; all stores have their handlers called for actions they share). This also supports `myReactor.someAction()`.